js.go

```diff
@@ -1581,7 +1636,7 @@ func (sub *Subscription) resetOrderedConsumer(sseq uint64) {
 		// Reset some items in jsi.
 		jsi.dseq = 1
 		jsi.cmeta = _EMPTY_
-		jsi.fcr, jsi.fcd = _EMPTY_, 0
+		jsi.fcr, jsi.fcd, jsi.fciseq = _EMPTY_, 0, 0
```

Comment:
I just made that change (with push force) after submitting the PR but I think it is wrong, since the subscription is maintained, the sub.delivered count is not reset either, so this value should not be reset. Will revert this change and push force again.